testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/redshift/BaseRedshiftFailureRecoveryTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.faulttolerant.redshift;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.operator.RetryPolicy;
+import io.trino.plugin.exchange.filesystem.FileSystemExchangePlugin;
+import io.trino.plugin.jdbc.BaseJdbcFailureRecoveryTest;
+import io.trino.testing.QueryRunner;
+import io.trino.tpch.TpchTable;
+
+import java.util.List;
+import java.util.Map;
+
+import static io.trino.plugin.redshift.RedshiftQueryRunner.createRedshiftQueryRunner;
+
+public abstract class BaseRedshiftFailureRecoveryTest
```

Comment:
Could you confirm CI failure? 
```
Error:  io.trino.faulttolerant.redshift.TestRedshiftQueryFailureRecoveryTest.init  Time elapsed: 12.943 s  <<< FAILURE!
java.lang.NoClassDefFoundError: io/trino/plugin/redshift/RedshiftPlugin

Error:  io.trino.faulttolerant.redshift.TestRedshiftTaskFailureRecoveryTest.init  Time elapsed: 0.009 s  <<< FAILURE!
java.lang.NoClassDefFoundError: io/trino/plugin/redshift/RedshiftPlugin
```

Reply:
Thank you. Verified fix and ran tests locally, can you re-test?

Reply:
Sure, just updated https://github.com/trinodb/trino/pull/16968